binary/integer.go

```diff
@@ -28,38 +28,34 @@ var tt255 = new(big.Int).Lsh(big1, 255)
 
 //--------------------------------------------------------------------------------
 
-func PutUint64LE(dest []byte, i uint64) {
-	binary.LittleEndian.PutUint64(dest, i)
-}
-
-func GetUint64LE(src []byte) uint64 {
-	return binary.LittleEndian.Uint64(src)
-}
-
-func PutUint64BE(dest []byte, i uint64) {
+func PutUint64(dest []byte, i uint64) {
```

Comment:
If I see a uint64 to bytes conversion, I would like to know what endianness it is. Then again most of these functions are a thin wrapper around encoding/binary so maybe they should go away instead.

Reply:
Yeah possibly...

My thinking here was that insofar as the binary package does anything it establishes a convention that if we just want some bytes for for an integer then it is big endian. If you want/need to think about endianess then you should be using binary directly.

They do provide a little bit of convenience around directly returning the bytes, casting etc. I _think_ I'd rather have them here than strewn around the rest of code, but open to persuasion.

Reply:
"establishes the convention" just means it's confusing for anyone new to our code and does not improve readability.

Reply:
let's remove the binary helpers next time we touch this